### PR TITLE
[LOOP-5158] Implement page objects and add extensions

### PR DESCRIPTION
**Pull request by Petr-Zywczok**

### CODE REVIEW

**Comment by Camji55** [comment]
@Petr-Zywczok this looks good to me! Same concern here as https://github.com/tidepool-org/TidepoolSupport/pull/61#issuecomment-2529971850 so let's hold off on merging for the moment.

File: Sources/LoopUITestingKit/Helpers/Utils.swift [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+            else {
+                let coordinate: XCUICoordinate = coordinate(withNormalizedOffset: CGVector(dx:0.0, dy:0.0))
+                coordinate.tap()
+            }
+        }
+    }
+}
+
+extension XCUIApplication {
+    public func uninstall(name: String? = "Tidepool Loop") {
```
Comment (Camji55) [inline Sources/LoopUITestingKit/Helpers/Utils.swift]: Not sure this would work with Tidepool Loop Coastal.

**Review by Camji55 — APPROVED** [review]
Looks good to me but please make sure these work on both Tidepool flavors of Loop.

File: Sources/LoopUITestingKit/Helpers/Utils.swift (hunk 1 above)
Comment (Petr-Zywczok) [reply Sources/LoopUITestingKit/Helpers/Utils.swift]: You are right. I have removed the default value.

**Review by nhamming — APPROVED** [review]
LGTM (considering cameron's comment above)

**Pull request merged**